lib/process-services/src/lib/form/widgets/content-widget/attach-file-widget.component.spec.ts

```diff
@@ -189,6 +194,17 @@ describe('AttachFileWidgetComponent', () => {
         fixture.destroy();
     });
 
+    const isPreviewButtonDisabled = async (fileId: string): Promise<boolean> => {
+        const menu = await loader.getHarness(MatMenuHarness);
+        await fixture.debugElement.query(By.css(`#file-${fileId}-option-menu`)).nativeElement.click();
```

Comment:
instead of using fixture.debugElement directly please use UnitTestingUtils class (there is clickByCSS function).

Reply:
done